spec/traits.dd

```diff
@@ -884,6 +885,13 @@ static assert(__traits(getLinkage, FooCPPInterface) == "C++");
 ---
 )
 
+$(H2 $(GNAME getLocation))
+        $(P Takes one argument which is a symbol.
+        To disambiguate between overloads, pass the result of getOverloads with the desired index, to getLocation.
```

Comment:
Surround code with tick marks:  getOverloads, getLocation.

Reply:
As Mike said, but also link to `getOverloads`.
```suggestion
        To disambiguate between overloads, pass the result of $(GLINK getOverloads) with the desired index, to `getLocation`.
```